cmd/upgrade/main.go

```diff
@@ -0,0 +1,46 @@
+package upgrade
+
+import (
+	"fmt"
+
+	"github.com/datreeio/datree/pkg/cliClient"
+	upgrademanager "github.com/datreeio/datree/pkg/upgradeManager"
+	"github.com/spf13/cobra"
+)
+
+type Printer interface {
+	PrintMessage(messageText string, messageColor string)
+}
+type UpgradeCommandContext struct {
+	CliVersion       string
+	Printer          Printer
+	UpgradeCliClient *cliClient.CliClient
+}
+
+func New(ctx *UpgradeCommandContext) *cobra.Command {
+	m := upgrademanager.NewUpgradeManager()
```

Comment:
Because this is an I/O dependency, adding it to the `UpgradeCommandContext` and injecting it from the outside in cmd/root.go makes more sense to me.
It will allow for easier testing